<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cookwindow</class>
 <widget class="QMainWindow" name="cookwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>点都德++餐厅管理与服务系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabBar::tab{
background: rgb(188, 203, 255);
	color: rgb(85, 85, 127);
	padding:20px;
width:293px;
height:30px;
	font: 14pt &quot;微软雅黑&quot;;
}
QTabBar::tab:selected{
background-color: rgb(189, 232, 255);

}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>700</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>任务清单</string>
     </attribute>
     <widget class="QTableWidget" name="tdTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>960</width>
        <height>450</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="bRefresh1">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>520</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string> 刷新列表</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bAdd">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>520</y>
        <width>220</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  加入“我的任务”</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>  我的任务</string>
     </attribute>
     <widget class="QTableWidget" name="myTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>960</width>
        <height>450</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="bComplete">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>530</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>    任务完成！</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>已完成列表</string>
     </attribute>
     <widget class="QTableWidget" name="cTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>960</width>
        <height>450</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="bRefresh2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>520</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string> 刷新列表</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bLogout">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>520</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  退出登录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bExit">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>520</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  退出系统</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
